bundles/auth/org.eclipse.smarthome.auth.oauth2client/src/main/java/org/eclipse/smarthome/auth/oauth2client/internal/OAuthClientServiceImpl.java

```diff
@@ -26,6 +26,7 @@
 import org.eclipse.jdt.annotation.NonNull;
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.util.StringUtil;
 import org.eclipse.jetty.util.UrlEncoded;
```

Comment:
I know, this is not part of this PR, but I was just wondering, why we have a Jetty dependency here - could `URLEncoder` be used instead?

Reply:
`URLEncoder` returns a complete string. Here is seems `UrlEncoded` is used to get easy access to parts of the url.